src/clj/medusa/alert.clj

```diff
@@ -33,18 +33,23 @@
          metric_id :id} (db/get-metric metric_id)
         {detector_name :name} (db/get-detector detector_id)
         subscribers (db/get-subscribers-for-metric metric_id)
-        alert-url (str "http://" hostname "/index.html#/detectors/" detector_id "/metrics/" metric_id "/alerts/?from=" date "&to=" date)]
+        alert-url (str "http://" hostname "/index.html#/detectors/" detector_id "/metrics/" metric_id "/alerts/?from=" date "&to=" date)
+        evo-url (str "http://telemetry.mozilla.org/new-pipeline/evo.html#!measure=" metric_name "&product=Firefox&start_date=" date "&end_date=" date "&table=0&use_submission_date=0")]
```

Comment:
I think we'd prefer `https` here.. and I don't know that evo uses start date -or- end date, come to think of it. Pretty sure it only knows channels and versions.